src/main/java/com/conveyal/r5/analyst/ResultSet.java

```diff
@@ -230,6 +235,32 @@ public void writeIsochrones (JsonGenerator jgen) throws IOException {
         jgen.writeEndArray();
     }
 
+    public void writeIsochrones(List<GeoJsonFeature> features, boolean returnDistinctAreas) {
```

Comment:
Unlike the other function called writeIsochrones, I don't see this function writing any JSON. It appears to just create GeoJSON features and put them into the list that was passed in. It also may not be ideal to structure a function to have side effects on a collection passed in as a parameter, when it's still possible to return values from that function (void return type).